crate/operator/restore_backup.py

```diff
@@ -74,6 +75,11 @@
 DEFAULT_MAX_BYTES_PER_SEC: str = "40mb"
 DEFAULT_CLUSTER_CONCURRENT_REBALANCE: int = 2
 CRASH_COMMAND_DELAY: int = 30
+RESTORE_TYPE_TABLES: str = "tables"
```

Comment:
Shouldn't this be an `Enum`, perhaps?

Reply:
yes, good point. thanks! I've changed it.